src/main/kotlin/uk/gov/justice/digital/hmpps/hmppselectronicmonitoringdataplatformapi/model/LocationAggregation.kt

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.digital.hmpps.hmppselectronicmonitoringdataplatformapi.model
+
+import java.util.*
+
+interface LocationAggregation {
+  var latitude: Double
+  var longitude: Double
+  var datetime: Date
+  fun setValue(latitude: Double, longitude: Double, datetime: Date)
+}
+
+class LocationAggregationDemo(override var latitude: Double, override var longitude: Double, override var datetime: Date) : LocationAggregation {
```

Comment:
Is there a reason this is called 'demo'? We should probably rename this to imply it's functional, even if functionality has limits.

Reply:
On second look, I don't know that this appears to be called anywhere - do we need it, or can we delete it (and the setValue function it uses on line 9)